spec/view_models/home_view_spec.rb

```diff
@@ -24,12 +25,17 @@
       let!(:product3) { FactoryBot.create :product, name: 'Product3', category: tshirts_category }
       let!(:product4) { FactoryBot.create :product, name: 'Product4', category: tshirts_category }
       let!(:product5) { FactoryBot.create :product, name: 'Product5', category: jeans_category }
-      let!(:product6) { FactoryBot.create :product, name: 'Product6', category: jeans_category }
-      let!(:product7) { FactoryBot.create :product, name: 'Product7', category: jeans_category }
-      let!(:product8) { FactoryBot.create :product, name: 'Product8', category: jeans_category }
 
-      it 'should return the last three women products' do
-        expect(home_view.new_products_women).to eq([product6, product7, product8])
+      let!(:concrete1) { FactoryBot.create :concrete_product, product: product1, price: 106.00, size: 'S', color: '#000000', amount: 7 }
+      let!(:concrete2) { FactoryBot.create :concrete_product, product: product1, price: 105.00, size: 'S', color: '#000000', amount: 6 }
+      let!(:concrete3) { FactoryBot.create :concrete_product, product: product2, price: 104.00, size: 'S', color: '#000000', amount: 5 }
+      let!(:concrete4) { FactoryBot.create :concrete_product, product: product3, price: 103.00, size: 'S', color: '#000000', amount: 4 }
+      let!(:concrete5) { FactoryBot.create :concrete_product, product: product2, price: 102.00, size: 'S', color: '#000000', amount: 3 }
```

Comment:
Jeśli parametry nie są istotne dla testu (cena, rozmiar, kolor), ale są wymagane do utworzenia obiektu to powinny być zdefiniowane w Factory, a tutaj już nie trzeba ich powtarzać. Będziemy mieli dzięki temu mniej kodu.